### ISSUE TITLE
Enable option to define a target class

### ISSUE THREAD

**Issue by JKutscha** [pitest/pitclipse#164]
I would like to add an option to the run configuration, where the user could define specific one or more classes which to mutate.
The option is implemented in pit, but we fill it with the complete class path.

This would allow faster testing in a big project, if only a few classes should be tested.

I would start implementing a simple text field, which could be enabled in the run configuration and later on I would want to add the functionality like the Type finder of eclipse, where the user begins to type and eclipse gives options to choose.

**Comment by LorenzoBettini** [comment]
Yes, I guess that would be useful: currently, you can only exclude classes. How would your proposal fit with the current run configuration? Would that live together with the current "Excluded classes" text field?

**Comment by JKutscha** [comment]
Yes, just like in PIT.

If you are using it via CLI, you need to define both, but currently we just use the classpath for the target classes.

I would make it optional to use the old system by default and only use it, if the user activates it.

**Comment by LorenzoBettini** [comment]
Closed by #197
